jax_galsim/exponential.py

```diff
@@ -46,10 +46,6 @@ def __init__(self, half_light_radius=None, scale_radius=None, flux=1.0, gsparams
         else:
             super().__init__(scale_radius=scale_radius, flux=flux, gsparams=gsparams)
 
-        self._r0 = self.scale_radius
```

Comment:
I'm confused, where are these parameters stored then?

Reply:
we made them all into properties, so they are not computed at the time of `__init__`